ci/ci.sh

```diff
@@ -45,10 +45,10 @@ kubectl get all --all-namespaces
 sleep 10
 cd $USERSPACEDIR
 
-docker build . -f ./docker/userspacecni/Dockerfile -t userspacecni:latest
+make build
 # gets path for one directopry above, needed for mkdir with docker cp below
 mkdir_var=$(dirname ${USERSPACEDIR})
-kind load docker-image userspacecni
+kind load docker-image localhost:5000/userspacecni
```

Comment:
should we specify the address of the local registry?
maybe make it a configurable var at the top and default to localhost?

Reply:
I guess since it's for CI, it can stay.